### PR TITLE
chore(RTL): install rtl related dev dependencies

### PR DESCRIPTION
**Pull request by mkholjuraev**
# Description

Associated Jira ticket: # (issue)

Installs rtl related packages 

**BLOCKER: this is needed to allow us to write RTL tests to existing PRs**
# How to test the PR

Please include steps to test your PR.

# Before the change

# After the change

# Dependent work link

# Checklist:

- [ ] The commit message has the Jira ticket linked
- [ ] PR has a short description
- [ ] Screenshots before and after the change are added
- [ ] Tests for the changes have been added
- [ ] README.md is updated if necessary
- [ ] Needs additional dependent work

### CODE REVIEW

**Review by bastilian — APPROVED** [review]
LGTM! Thank you @mkholjuraev!

File: config/setupTests.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+jest.mock('react', () => ({
+  ...jest.requireActual('react'),
+  useLayoutEffect: jest.requireActual('react').useEffect,
+}));
```
Comment (bastilian) [inline config/setupTests.js:4]: Why this?

**Review by bastilian — CHANGES_REQUESTED** [review]
That override of useLayoutEffect seems dangerous.

File: config/setupTests.js (hunk 1 above)
Comment (mkholjuraev) [reply config/setupTests.js:4]: aha will remove this of course. it is not needed

**Review by bastilian — APPROVED** [review]
Now it looks good! :) Thank you @mkholjuraev!

**Pull request merged**